src/Microsoft.Azure.WebJobs.Host/Extensions/ITooling.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Reflection;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// Tooling interface for script 
+    /// </summary>
+    public interface ITooling
```

Comment:
Maybe I'm the only one, but when I see Tooling, I think of Visual Studio. Is there a different name we could give this to better reflect what it's for?